pod-security/baseline/disallow-adding-capabilities/disallow-adding-capabilities.yaml

```diff
@@ -8,7 +8,8 @@ metadata:
     policies.kyverno.io/subject: Pod
     policies.kyverno.io/description: >-
       Capabilities permit privileged actions without giving full root access.
-      Adding capabilities beyond the default set must not be allowed.
+      Adding capabilities beyond the default set must not be allowed. This sample
```

Comment:
Instead of using `this sample` I suggest we use `this policy`.  The reason is to make it easier for teams to clone, kustomize, and use these policy definitions without having to rewrite the descriptions for their devs.

I realize this impacts a number of policies, but hopefully is mostly a search and replace.

In some cases, if the policy is not ready for direct use, its fair to call it a sample.